ca_on_toronto/committees.py

```diff
@@ -4,8 +4,56 @@
 from .helpers import build_lookup_dict, committees_from_sessions
 from .constants import TWO_LETTER_ORG_CODE_SCHEME
 
+import re
+
+
+MEMBERS_URL_TEMPLATE = 'http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doGetMembers&meetingId={}&showLink=true'
+DEFAULT_COMMITTEE_ROLE = 'Member'
+
+REFERENCE_MEETING_IDS = {
```

Comment:
Yeah, it's not ideal, but it works for this term, and I can foresee how to improve it once the end of the term nears.

Basically:
* members by decisionBodyId are *sometimes* available, but not always, but lack the role information: (I also can't figure out the rhyme or reason for when this links works and when it 404s)
http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doGetMembers&decisionBodyId=1032&showLink=true
* meetingId seems to have more complete information:
http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doGetMembers&meetingId=11014&showLink=true
* but meetingId urls vary in time -- you can see this by expanding meeting links here:http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doPrepare&decisionBodyId=1181#Meeting-2016.AU5 (future dates seem to lack role data, and past dates sometimes lack persons or roles, almost like it's a reflection of attendance)

So basically, it seems attendance data is inconsistent in ways that no one has clarified to me yet. And so I was instead opting for a representative meeting.

In the future, we could add a means to request each meeting page (app.toronto.ca/tmmis/decisionBodyProfile.do?function=doPrepare&decisionBodyId=1181) and follow the redirect in order to parse the most recent meeting from the fragment identifier (http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doPrepare&decisionBodyId=1181#Meeting-2016.AU5), and then hit that link. But it seemed a bit more complicated, and so this seemed a better route for now?